multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -108,6 +111,8 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 	r.mapLock.Lock()
 	defer r.mapLock.Unlock()
 
+	// If !ok, it means member not found. If this happens, the MemberClusterAnnounce should soon be deleted.
```

Comment:
I checked the code. I think we need to revise the comment to: "If the member is not found in timerData, it means the member is not added to the ClusterSet or was to be deleted."

Reply:
Revised.